Worksheet_B/README.md

```diff
@@ -0,0 +1,17 @@
+Print “ Welcome to Terminal 3 ”
+Print “ Password Required “
+Secret word = Pool
+Player strikes left: = 4
+Computer generates 14 words 4 letter words in random order
+Computer words = (“Pool, Roam, Seek, Dean, Foes, Half, Word, Copy, Cold, Fend, Wins, Pick, Died, Menu”)
+If player word is not equal to secret word player loses a strike
+	If letter = same position is Secret Word display similarity
```

Comment:
The worksheet asked you to provide pseudocode for calculating the similarity score, e.g. by looping over the words and counting the number of matching letters.